nodes/nomos-node/config.yaml

```diff
@@ -201,6 +201,9 @@ da_sampling:
   network_adapter_settings: null
   storage_adapter_settings:
     blob_storage_directory: ./
+  api_adapter_settings:
+    addresses:
+      - http://127.0.0.1:8722
```

Comment:
Now that I see this. This should be a membership in the same fashion we have in the da-network-backend, as it need to be shared and updated. Can we add that?